README.Rmd

```diff
@@ -122,6 +123,28 @@ imeta(
 # check if file is stored with associated metadata
 ils(metadata = TRUE)
 ```
+#### Reading metadata
+Assume you have a well-annotated object and you would like to use some of the metadata in your analysis. how can you read the metadata of "foo" into an R table?
+In the code below we extract the metadata table which the `ils` command provides us with and assign it to a new variable we can further work with:
+
+```r
+foo_meta = ils(path="/bobZone/home/christine/test/foo", metadata = TRUE)
```

Comment:
Not a blocker: The other examples seem to be using relative paths, so I'm wondering if we should be consistent and do that here as well.

Reply:
Take care that object are assigned with `<-` by convention, except for function parameters. Also, the absolute path will make rendering the README.rmd more difficult. The other examples are best on the irods_demo.